### PR TITLE
680: Add new view to caseload overview at team level.

### PR DESCRIPTION
**Pull request by dconey646**

### CODE REVIEW

File: seed/views/00901_team_case_overview.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  AS
+  SELECT
+      MAX(CONCAT(om.forename, ' ', om.surname)) AS name
+    , MAX(om.grade_code) AS grade_code
+    , MAX(w.total_cases) AS total_cases
+    , MAX(wpc.available_points) AS available_points
+    , MAX(wpc.total_points) AS total_points
+    , MAX(wpc.contracted_hours) AS contracted_hours
+    , MAX(wpc.reduction_hours) AS reduction_hours
+    , MAX(t.id) AS team_id
```
Comment (tsheppard01) [inline seed/views/00901_team_case_overview.js]: The case-progress views use `id` rather that `team_id`, `region_id` etc. This allows the get-case-progress to become more generic.  Can you change this to use `id` as well.

Comment (dconey646) [inline seed/views/00901_team_case_overview.js]: Good spot. I will get that in.

File: seed/views/00901_team_case_overview.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+exports.seed = function (knex, promise) {
+  var sql = `CREATE VIEW app.team_case_overview
```
Comment (tsheppard01) [inline seed/views/00901_team_case_overview.js]: Should this be called team_case_overview_view?

**Comment by tsheppard01** [comment]
LGTM 👍

**Pull request merged**